packages/core/src/interpreter.ts

```diff
@@ -377,12 +319,20 @@ export class Interpreter<
     this.stopListeners.add(listener);
     return this;
   }
+
+  /**
+   * Adds an error listener that is notified with an `Error` whenever an
+   * error occurs during execution.
+   *
+   * @param listener The error listener
+   */
   public onError(
```

Comment:
isnt this also somewhat redundant? one could just subscribe to error listeners via subscribe

Reply:
No, this should be for any errors, even for those "uncaught" by transitions.